doc/cephfs/add-remove-mds.rst

```diff
@@ -62,7 +62,15 @@ means limiting its cache size.
 Adding an MDS
 =============
 
-#. Create an mds data point ``/var/lib/ceph/mds/ceph-${id}``. The daemon only uses this directory to store its keyring.
+NOTE: It is highly recommended to use `cephadm`_ for setting up the ceph
```

Comment:
Correct markdown for this is:

`.. note:: the message`

See other places in docs for examples.

I think you should also update the introductory text of this doc to mention cephadm.

Reply:
I've corrected the markdown for the `note`, thanks! 

I think you should also update the introductory text of this doc to mention cephadm.
> You mean to say I mention cephadm in the "DEPLOYING METADATA SERVERS" section?

Reply:
yes